Documentation/operation docs rework (#3074)
scality/metalk8s: docs/operation/troubleshooting/index.rst

.. _Troubleshooting Operation Guide:

Troubleshooting
===============

> This guide covers some of the common issues users face during and

Reviewer: ```suggestion
This section covers common issues users face during and
```
